docs: Verbeteren en uitbreiden Community Sprints pagina #2213 (#2288)
nl-design-system/documentatie: docs/community/community-sprints/overheidsbrede-portalen-community.mdx

---
title: Overheidsbrede Portalen Community · Community Sprints · Community
hide_title: true
hide_table_of_contents: false
sidebar_label: Overheidsbrede Portalen Community
pagination_label: Overheidsbrede Portalen Community
description: Informatie over de NL Design System Overheidsbrede Portalen Ccmmunity Sprint
keywords:
  - nl design system
  - community sprints
  - overheidsbrede portalen community
---

{/* @license CC0-1.0 */}

import { VideoPlaylistPlayer } from "@site/src/components/VideoPlaylistPlayer";

# Community Sprint: Overheidsbrede Portalen Community

> Vanaf September 2024 is de tweede NL Design System Community Sprint van start gegaan. Dit keer werken teams samen aan het onderwerp Overheidsbrede portalen. [VNG](https://vng.nl) werkt hiervoor samen met [Ministerie van Justitie en Veiligheid](https://www.rijksoverheid.nl/ministeries/ministerie-van-justitie-en-veiligheid).

Reviewer: We hebben tijdens de Mijn Mijn Mijn workshop met een heleboel gemeenten en organisaties samengewerkt om te komen dit de minimale standaar templates...

> Vanaf September 2024 is de tweede NL Design System Community Sprint van start gegaan. Dit keer werken teams samen aan het onderwerp Overheidsbrede portalen. [VNG](https://vng.nl) werkt hiervoor samen met [Ministerie van Justitie en Veiligheid](https://www.rijksoverheid.nl/ministeries/ministerie-van-justitie-en-veiligheid).

Author: ```suggestion
Vanaf september 2024 is de tweede NL Design System Community Sprint van start gegaan. Dit keer werken teams samen aan het onderwerp Overheidsbrede portalen. Dit is een samenwerking van onder andere BZK, Logius/MijnOverheid, [VNG](https://vng.nl/omnichannel), [Ministerie van Justitie en Veiligheid](https://www.rijksoverheid.nl/ministeries/ministerie-van-justitie-en-veiligheid), [Gemeente Den Haag](https://github.com/nl-design-system/denhaag), NLPortal, OIP en OpenPDD. Deze partijen zijn deze ontwerpen en best practices aan het implementeren of beproeven.
```

## Waar gaat deze sprint over?

Een heldere, tijdloze uitleg in 2 a 3 alinea's:

- Wat is het onderwerp?
- Wat zijn de belangrijkste vragen of uitdagingen die hierin centraal staan?
- Waarom is dit relevant binnen het NL Design System of voor overheidsorganisaties?

> Binnen het onderwerp Overheidsbrede Portalen maken we gebruik van de kracht van het NL Design System: een gezamenlijke inspanning van specialisten uit diverse overheidsorganisaties om toegankelijke, gebruiksvriendelijke en consistente digitale oplossingen te realiseren met de. In deze Community Sprint ontwikkelen we aan een uitgebreide set van white-label templates die direct toepasbaar zijn voor websites en applicaties binnen de overheid.

Author: ```suggestion
In deze Community Sprint ontwikkelen we aan een uitgebreide set van huisstijl-onafhankelijke templates voor Mijn Omgevingen en Formulieren die direct te gebruiken zijn voor websites en applicaties binnen de overheid.
```

### Templates

> Elke template biedt een sterke basis voor het creëren van verschillende soorten pagina’s binnen je project. Ze zijn ontworpen met focus op schaalbaarheid en een intuïtieve gebruikerservaring. Of je nu werkt aan een informatieve pagina, een interactief formulier of een specifieke gebruikersflow, deze templates helpen je om snel tot een betrouwbaar resultaat te komen. De structuur en layout van de templates zijn gestandaardiseerd, wat zorgt voor consistentie en efficiëntie in de ontwikkeling.

Reviewer: (Ter info: Wat Brian zegt is wat we tijdens de Mijn Mijn Mijn Workshop hebben gedaan)

> Elke template biedt een sterke basis voor het creëren van verschillende soorten pagina’s binnen je project. Ze zijn ontworpen met focus op schaalbaarheid en een intuïtieve gebruikerservaring. Of je nu werkt aan een informatieve pagina, een interactief formulier of een specifieke gebruikersflow, deze templates helpen je om snel tot een betrouwbaar resultaat te komen. De structuur en layout van de templates zijn gestandaardiseerd, wat zorgt voor consistentie en efficiëntie in de ontwikkeling.

Author: ```suggestion
Gebruik de templates van de Overheidsbrede Portalen Community als een sterke basis voor verschillende soorten pagina’s binnen je project. De templates zijn ontworpen met focus op schaalbaarheid en een intuïtieve gebruikerservaring. Of je nu werkt aan een pagina, een formulier of een specifieke flow, met deze templates werken we toe naar een éénduidige beleving voor inwoners en ondernemers.
```

## Werk en resultaten

Bekijk de links en documentatie die horen bij deze Community Sprint:

- [Overheidsbrede Portalen Community Repository op GitHub](https://github.com/nl-design-system/overheidsbrede-portalen-community)
- [Overheidsbrede Portalen Community Sprintbord op GitHub](https://github.com/orgs/nl-design-system/projects/34)
- [NL Design System - Overheidsbrede Portalen Community - Templates](https://www.figma.com/design/iZgSIuU8hvH9nw3h7WO1ZY/Overheidsbrede-portalen---Templates?node-id=1-3&p=f&t=Y6E4Abu0nUUgDpYa-0)
- [Overheidsbrede Portalen Community Storybook](https://nl-design-system.github.io/overheidsbrede-portalen-community/)

Bekijk hieronder de YouTube afspeellijst met de video's van de Overheidsbrede Portalen Community.

<VideoPlaylistPlayer playlistId="PLnAxFMscDU3F1VYQxlJLpmV-Z-FQ6hCU_" />

## Voor wie is dit relevant?

> Deze sprint is bedoeld voor iedereen die werkt aan TODO:, en in het bijzonder voor:

Reviewer: of "Mijn omgevingen"

> Deze sprint is bedoeld voor iedereen die werkt aan TODO:, en in het bijzonder voor:

Author: ```suggestion
Deze sprint is bedoeld voor iedereen die werkt aan portalen, Mijn Omgevingen en Formulieren, en in het bijzonder voor:
```

- [Overheidsbrede Portalen Community Sprintbord op GitHub](https://github.com/orgs/nl-design-system/projects/34)
- [NL Design System - Overheidsbrede Portalen Community - Templates](https://www.figma.com/design/iZgSIuU8hvH9nw3h7WO1ZY/Overheidsbrede-portalen---Templates?node-id=1-3&p=f&t=Y6E4Abu0nUUgDpYa-0)
- [Overheidsbrede Portalen Community Storybook](https://nl-design-system.github.io/overheidsbrede-portalen-community/)

Bekijk hieronder de YouTube afspeellijst met de video's van de Overheidsbrede Portalen Community.

<VideoPlaylistPlayer playlistId="PLnAxFMscDU3F1VYQxlJLpmV-Z-FQ6hCU_" />

## Voor wie is dit relevant?

Deze sprint is bedoeld voor iedereen die werkt aan portalen, Mijn Omgevingen en Formulieren, en in het bijzonder voor:

- Designers die willen bijdragen aan een gedeelde visuele stijl
- Developers die componenten implementeren volgens Overheidsbrede Portalen
- Product Owners die keuzes maken over ontwerpstandaarden
- Iedereen die bijdraagt aan een toegankelijk, samenhangend digitaal landschap
- …

## Hoe kun je bijdragen of meedoen?

### Meepraten of input geven?

> Je kunt ons ook vinden op Slack. Meld je aan bij onze [Slackgroep ‘NL Design System (praatmee.codefor.nl)’](https://praatmee.codefor.nl/) en sluit aan bij #nl-design-system.

Author: ```suggestion
Je kunt ons ook vinden op Slack. Meld je aan bij onze [Slackgroep ‘NL Design System (praatmee.codefor.nl)’](https://praatmee.codefor.nl/) en sluit aan bij #nl-design-system.

Ook kan je elke sprint meedoen aan de Overheidsbrede Portalen Community Open Hour en de thread volgen in het #nl-design-system-developer kanaal.
```
